pkg/Library/Fields/Atoms/SelectField.js

```diff
@@ -37,21 +31,19 @@ template: html`
   [label] {
     background: inherit;
     font-weight: bold;
-    /* font-size: 75%; */
     border: none;
-    text-align: left;
-    /* width: 16em; */
+    text-align: right;
+    min-width: 8em;
   }
   [delim] {
     padding-right: 12px;
   }
 </style>
 
 <div flex bar>
-  <!-- <input label value="{{label}}" on-change="onLabelChange"> -->
   <div label>{{label}}</div>
-  <span delim>:</span>
-  <multi-select flex field options="{{options}}" selected="{{value}}" selected="{{value}}" multiple="{{multiple}}" on-change="onFieldChange"></multi-select>
+  <span delim></span>
```

Comment:
thank you, this was bugging me for a long time (now i don't know why i never removed it 🤦 )